Import/HideTabsByIdentity/Import/Method/labs_HideTabsByIdentity.xml

```diff
@@ -7,127 +7,63 @@ var inn = document.thisItem.getInnovator();
 
 //Function to hide tabs
 hideTabs = function() {
-
     // check whether current user is a member of the target identity
-    // get user info
-    var userID = top.aras.getCurrentUserID();
-    var user = inn.getItemById("User", userID);
-    var userName = user.getProperty("keyed_name", "");
-
-    // get target identity
-    var targetIdent = inn.newItem("Identity", "get");
-    targetIdent.setProperty("keyed_name", "Administrators");
-    var memrel = targetIdent.createRelationship("Member", "get");
-    targetIdent = targetIdent.apply();
-
-    if (targetIdent.isError()) {
-        return targetIdent;
-    }
-
-    var isMember = memberCheck(targetIdent, userName, 0);
+    var isMember = memberCheck("Administrators");
 
     // if user is a member of target identity, return. otherwise, hide tabs
-    if (isMember == 1)
+    if (isMember != -1)
```

Comment:
```suggestion
    if (isMember)
```
The current line is a bit of a double negative asking if the user is not not a member of the identity. Combined with the suggested change below, we can make this a bit more readable.